olive/passes/onnx/model_builder.py

```diff
@@ -137,6 +178,19 @@ def _run_for_config(
             model_path = str(model.hf_config.model_name)
             input_path = ""
 
+        if config.get("int4_block_size"):
+            if int(config["int4_block_size"]) not in [16, 32, 64, 128, 256]:
+                raise RuntimeError("Invalid int4_block_size. Accepted values: 16/32/64/128/256.")
+            extra_args["int4_block_size"] = config["int4_block_size"]
+        if "int4_accuracy_level" in config:
```

Comment:
I don't think this and later if checks are necessary. The pass run always gets a full config so the keys will always be present.

Reply:
Actually it applies to the Boolean parameters.

For the accuracy and block size, the check should be for None since the keys are always present.

Reply:
Done!